### DISCUSSION TITLE
YOURLS IP Viewer plugin not listed here?

### DISCUSSION THREAD

**Discussion by HeadOfTheSPEDs** [YOURLS/YOURLS#3423] [Plugins]
I've been making use of the YOURLS IP Viewer plugin by @faab007nl located at https://github.com/faab007nl/Yourls-Ip-Viewer and find it very useful. I don't recall how I found it but it's not listed on the Awesome YOURLS list. Should it be? If not, why isn't it listed? Just a suggestion...

I'm excited because I've just added functionality to it (about to post about it on that plugin's GitHub page) which will be my first ever actual contribution to GitHub - not bad for someone who doesn't really know how to code!

**Comment by dgw** [comment]
> why isn't it listed?

Because no one submitted it. Awesome YOURLS is populated by stuff people submit for inclusion.

**Reply by HeadOfTheSPEDs** [reply]
Ok, so how do I submit it?! Is it ok for me to submit it even though I didn't create it or officially contribute to it yet (https://github.com/faab007nl/Yourls-Ip-Viewer/issues/1)? Sorry, I'm very new at this!

**Reply by dgw** [reply]
https://github.com/YOURLS/awesome-yourls/blob/main/CONTRIBUTING.md
